shell/lint.sh

```diff
@@ -31,7 +31,7 @@ then
   exit 1
 fi
 [ $# -eq 0 ] && echo "no issues with flake8"
-black --check $files
+black --line-length 80 --check $files
```

Comment:
I suggest to move this in the configuration file or we misaling IDE integrations.

Reply:
@bhack I like your suggestion but let's save this for a followup PR. We need to find a unified solution that works across all `keras-team` repos long term.